smartsim/entity/model.py

```diff
@@ -210,6 +218,10 @@ def colocate_db_uds(
         :type kwargs: dict, optional
         """
 
+        self.dbs_list_m(db_identifier)
```

Comment:
I can't tell from the variable name what this is for. Consider renaming it

Reply:
removed